### PR TITLE
Added a note on the value for inactive systems

### PR DESCRIPTION
**Pull request by mssola**
See SUSE/scc-docs#255.

### CODE REVIEW

**Comment by mssola** [comment]
[[URAS] Better document what 'inactive' means](https://trello.com/c/q72OrJao/2658-uras-better-document-what-inactive-means)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/287293/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged c562eee83d123d3b8627bf18bd847501e628eeed into 30ce37b3eca8fdd46385e6ece38da2db637b8d83)

File: lib/rmt/cli/systems.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class RMT::CLI::Systems < RMT::CLI::Base
-  # Amount of time for which a system is considered inactive.
+  # Amount of time for which a system is considered inactive. This is a
```
Comment (digitaltom) [inline lib/rmt/cli/systems.rb:2]:
```suggestion
  # Amount of time after which a system is considered inactive. This is a
```

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/287310/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged e9e178f6af8f3cb4a0f97934bddba9c3e4aa7832 into 30ce37b3eca8fdd46385e6ece38da2db637b8d83)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/287314/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged e9e178f6af8f3cb4a0f97934bddba9c3e4aa7832 into 28209a15ff19ced4a3e9d210ea90bb5b7c9fe777)

**Review by digitaltom — APPROVED** [review]

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/287315/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged 0224c47a46c5f6e861b352c0780faf989984b2c3 into 28209a15ff19ced4a3e9d210ea90bb5b7c9fe777)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/287324/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 0224c47a46c5f6e861b352c0780faf989984b2c3 into 28209a15ff19ced4a3e9d210ea90bb5b7c9fe777)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/scc-RMT-integration-tests/287325/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 36c706d44d2209db04fe9319f9e62715457c7c44 into 92e8241ac6a217d415d725f2ef53e71deecd7d2f)

**Pull request merged**